src/scss/partials/_variables.scss

```diff
@@ -7,7 +7,7 @@ $button-color: rgb(255, 175, 14);
 $letters:  rgb(194, 193, 192);
 
```

Comment:
Proměnné barev by měla být označení barvy a né toho kde se barva použije. $dark-blue, $light-blue, $red apod.

Nepoužívej názvy barev jako "white" působí to trošku amatérsky. Používej kód v hexadecimálním nebo rgb tvaru.

Reply:
upraveno